### ISSUE TITLE
TBG Feature request #588 (Draken on 2014-03-17): Profile based Overwrite - optional

### ISSUE THREAD

**Issue by TanninOne** [TanninOne/modorganizer#96]
It would help to have a profile based overwrite folder.

Current Setup: Profile1 and Profile2 use "overwrite" based in the Mod Organizer ''home dir''.

Request: Option to enable Profile based Overwrite folder for Profile2, before starting the Game exe MO copy's and replaces all files in Overwrite to a folder named "overwrite - profile2" based in ''Mod Directory'' and deletes them from Overwrite based in ''home dir''.

**Comment by TanninOne** [comment]
Freso on 2014-08-07: You can right-click the "''Overwrite''" 'mod' and create a 'proper mod' from it, which you can then have enabled/disabled across your profiles as you please. It would be nice with some more power/flexibility with how to handle the "''Overwrite''" mod, but I intend to make separate feature requests for that.

For some things, it's nice to have a global override (e.g., cleaned DLC .esm's).
